app/services/books/index.rb

```diff
@@ -10,6 +10,9 @@ class Index
 
     DEFAULT_INDEXING_STRATEGY = IndexingStrategies::I1::Strategy
 
+    RAP_URL_BASE = "https://openstax.org/apps/archive"
+    LEGACY_URL = "https://openstax.org/contents"
```

Comment:
These were hardcoded before as `https://archive.cnx.org`, not sure if we want them to be configurable?